frontend/src/modules/LoginRedux.js

```diff
@@ -1,27 +1,87 @@
 // Action Types
-export const SET_LOGIN_STATE = "localize/SET_LOGIN_STATE";
+// AUTH ACTIONS
+export const AUTHENTICATED = "AUTHENTICATED";
+export const UNAUTHENTICATED = "UNAUTHENTICATED";
+export const TESTING_TOKEN_REFRESH = "TESTING_TOKEN_REFRESH";
+export const REGISTRATION_SUCCESS_MESSAGE = "REGISTRATION_SUCCESS_MESSAGE";
+//CHANGE PASSWORD ACTIONS
+export const IS_CHANGING_PASSWORD = "IS_CHANGING_PASSWORD";
+export const CHANGE_PASSWORD_SUCCESS = "CHANGE_PASSWORD_SUCCESS";
+export const CHANGE_PASSWORD_FAILURE = "CHANGE_PASSWORD_FAILURE";
 
-// Action Creators
-const setLoginState = loggedIn => ({ type: SET_LOGIN_STATE, loggedIn });
+// Simplified version of the authentication action (temporary)
+const authenticateAction = authenticated => ({ type: AUTHENTICATED, authenticated });
+
+function registrationSuccessMessage() {
+  return { type: REGISTRATION_SUCCESS_MESSAGE };
+}
+
+function registerAction(data) {
+  return async function() {
+    let response = await fetch("api/auth/users/create/", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    });
+    let responseJson = response.json();
+    return responseJson;
+  };
+}
+
+function loginAction(data) {
+  return async function() {
+    let response = await fetch("/api/auth/jwt/create/", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    });
+    let responseJson = await response.json();
+    return responseJson;
+  };
+}
+
+// JWT tokens are not stored in our DB
+function logoutAction() {
+  localStorage.removeItem("ecom_token");
+  return { type: UNAUTHENTICATED };
+}
 
 // Initial State
 const initialState = {
-  loggedIn: false
```

Comment:
you are using `loggedIn` in your components, if it's required you need to keep it in the initial state.